package-system/DirectXShaderCompiler/build_dxc_windows.cmd

```diff
@@ -22,7 +22,7 @@ IF %ERRORLEVEL% NEQ 0 (
 )
 
 REM Make sure TAEF is installed
-call utils\hct\hctgettaef.py
```

Comment:
I think the %PYTHON_BINARY% define is declared by pull_and_build_from_git in all build scripts if you want this to work even without python in the path.   Its better than assuming that python is on the path, or that its the correct one (ie, might be python 27)

Reply:
Thanks for the feedback. I changed it to %PYTHON_BINARY%.